scripts/hist_importer.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# This file is part of qutebrowser.
```

Comment:
Please put a copyright notice above this, like in other files.

You can use this, or your own name:

```python
# Copyright 2017 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
#
# This file is part of qutebrowser.
# ...
```